hermetic_build/library_generation/cli/entry_point.py

```diff
@@ -30,18 +28,7 @@ def main(ctx):
 
 @main.command()
 @click.option(
-    "--baseline-generation-config-path",
-    required=False,
-    default=None,
-    type=str,
-    help="""
-    Absolute or relative path to a generation_config.yaml.
-    This config file is used for commit history generation, not library
-    generation.
-    """,
-)
-@click.option(
-    "--current-generation-config-path",
+    "--generation-config-path",
```

Comment:
I think this means our library generation would not need internet access anymore?

Reply:
I think so.